### PR TITLE
Update sharepoint-app-bar.md

### PR DESCRIPTION
**Pull request by petey121175**
I thought this was documented already as I remember seeing this somewhere. Currently, the App Bar is not available to external users of an organization. We have a couple reports of this and I was able to repro as well. Will send note to Holland too.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/sharepoint-app-bar.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ The SharePoint app bar brings together intranet resources and personalized conte
 > -	Specific SharePoint app bar tabs cannot be disabled. 
 > -	The SharePoint app bar cannot be disabled on specific sites. 
 > -	The SharePoint app bar is not available on classic SharePoint sites today, however soon administrators will be able to add it to classic sites manually. 
+> - The SharePoint app bar is not available to guest users outside of your organization.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sharepoint-app-bar.md:41]:
```suggestion
> -	The SharePoint app bar is not available to guest users outside of your organization.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor formatting suggestion.

**Review by scanum — APPROVED** [review]

**Comment by scanum** [comment]
@Holland-ODSP PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by Holland-ODSP — APPROVED** [review]

**Comment by Holland-ODSP** [comment]
Approved

**Pull request merged**